File: src/main/java/teoriadosgrafos/excecoes/CaminhoNaoEncontradoException.java
package teoriadosgrafos.excecoes;

/**
 * Exceção representando um caminho inalcançável partindo da origem ao destino
 */
public class CaminhoNaoEncontradoException extends RuntimeException {
	public CaminhoNaoEncontradoException( String s ) {
		super( s );
	}
}

File: src/main/kotlin/teoriadosgrafos/excecoes/CaminhoNaoEncontradoException.kt
package teoriadosgrafos.excecoes

/**
 * Exceção representando um caminho inalcançável partindo da origem ao destino
 *
 * @author [Icaro D Temponi](https://github.com/icarohs7)
 */
class CaminhoNaoEncontradoException(s: String) : RuntimeException(s)
